### PR TITLE
Response teams now show up in the orbit menu

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Response teams now show up in the orbit menu

Image of it in game:
image

## Why It's Good For The Game
ERTs, SSTs, and deathsquads are all pretty obviously impactful stuff, but you kind of need to fiddle around with the orbit menu for a bit looking for keywords to actually be able to orbit them, this streamlines this and I also like the color purple.

## Testing
Slammed my head into the wall for awhile trying to get the goddamn NodeJS to work but after a couple of reinstalls it functioned

## Changelog
:cl:
tweak: ERTs, SSTs, and deathsquads appear in the orbit menu under "Response teams"
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/observer/orbit.dm [hunk 1]
```diff
@@ -79,6 +80,9 @@
 				alive += list(serialized)
 
 				var/datum/mind/mind = M.mind
+				if(mind && (mind.special_role == SPECIAL_ROLE_ERT || mind.special_role == SPECIAL_ROLE_DEATHSQUAD || mind.special_role == SPECIAL_ROLE_SYNDICATE_DEATHSQUAD))
```
Comment (Sirryan2002) [inline code/modules/mob/dead/observer/orbit.dm:83]: don't need to check mind here, it's already null checked on L73
also
```suggestion
				if(mind.special_role in list(SPECIAL_ROLE_ERT, SPECIAL_ROLE_DEATHSQUAD, SPECIAL_ROLE_SYNDICATE_DEATHSQUAD)))
```

File: code/modules/mob/dead/observer/orbit.dm [hunk 2]
```diff
@@ -79,6 +80,9 @@
 				alive += list(serialized)
 
 				var/datum/mind/mind = M.mind
+				if(mind.special_role in list(SPECIAL_ROLE_ERT, SPECIAL_ROLE_DEATHSQUAD, SPECIAL_ROLE_SYNDICATE_DEATHSQUAD))
```
Comment (warriorstar-orion) [inline code/modules/mob/dead/observer/orbit.dm:83]: will this instantiate this list every time? if so you may want to pull it out of the for loop, or out of the proc altogether, though that might be overkill

Comment (warriorstar-orion) [reply code/modules/mob/dead/observer/orbit.dm:83]: discussed offline, probably overkill

**Review by warriorstar-orion — APPROVED** [review]
DM + JS LGTM

**Review by S34NW — APPROVED** [review]
Deconflict for merge please

**Pull request merged**